.github/workflows/run_doublet-detection.yml

```diff
@@ -65,15 +62,21 @@ jobs:
         with:
           working-directory: ${{ env.MODULE_PATH }}
 
-      - name: Set up conda
-        uses: conda-incubator/setup-miniconda@v3
+      # TODO: If/when the benchmarking code is added to this GHA, this conda environment setup needs to be uncommented
+      #
+      #- name: Set up conda
+      #  uses: conda-incubator/setup-miniconda@v3
+      #
+      #- name: Install conda-lock and activate locked conda environment
+      #  run: |
+      #    conda install conda-lock
+      #    conda-lock install --name test ${MODULE_PATH}/conda-lock.yml
 
-      - name: Install conda-lock and activate locked conda environment
-        run: |
-          conda install conda-lock
-          conda-lock install --name test ${MODULE_PATH}/conda-lock.yml
+      - name: Download test data
+        run: ./download-data.py --test-data --format SCE --project SCPCP000001
 
       - name: Run doublet-detection module
         run: |
           cd $MODULE_PATH
-          bash run_doublet-detection.sh
+
+          bash run_doublet-detection-scpca.sh SCPCP000001
```

Comment:
For best testing, we probably want to run on all projects. Probably something like:

```
projects=$(for p in data/current/SCPCP*; do basename $p; done)

cd $MODULE_PATH
for project in projects; do
  bash bash run_doublet-detection-scpca.sh $project
done
```

Of course, you will also want to download all of the test data as well.


In response to your other question, you may well want to use your Docker image to speed up test runs: see https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/.github/workflows/run_cell-type-ewings.yml for an example where @allyhawkins just tested setting this up: you will need to install `aws` unless that is already part of your Docker image. (Also note that the `defaults:` section can probably be deleted; I meant to tell @allyhawkins that during review, but forgot amid the other things we were trying to get running)